.gitignore

```diff
@@ -13,3 +13,14 @@ data.prototext*
 # Can also ignore all directories and files in a directory.
 # tmp/**/* 
 build
+spack_environments/users/
+
+# can ignore experiment/data directories
+applications/ATOM/charvae_data
+applications/ATOM/charvae_experiments
+
+# can ignore the moses repo
+applications/ATOM/moses/
```

Comment:
Not a big deal, but I'd recommend putting this inside a `.gitignore` file within `applications/ATOM`. That would keep it more self-contained.

Reply:
Seconded.

Reply:
Done